Dockerfile

```diff
@@ -1,42 +1,39 @@
+# Base Image
 FROM ubuntu:20.04
 ARG DEBIAN_FRONTEND=noninteractive
 
+
 WORKDIR /app
 
 RUN apt-get update -y && \
-  apt-get install build-essential -y && \
-  apt-get install git -y && \
-  apt-get install curl -y
-
-RUN apt-get install -y libgtk2.0-0 libgtk-3-0 libgbm-dev \
-  libnotify-dev libgconf-2-4 libnss3 libxss1 libasound2 \
-  libxtst6 xauth xvfb tzdata software-properties-common
+    apt-get install -y build-essential git curl \
+    libgtk2.0-0 libgtk-3-0 libgbm-dev libnotify-dev \
+    libgconf-2-4 libnss3 libxss1 libasound2 libxtst6 \
+    xauth xvfb tzdata software-properties-common
 
 RUN add-apt-repository ppa:deadsnakes/ppa -y && \
-  apt-get install python3.12 python3-pip -y && \
-  pip install pipenv
+    apt-get install -y python3.12 python3-pip && \
+    pip install pipenv
+
 
-RUN curl -sL https://deb.nodesource.com/setup_20.x -o nodesource_setup.sh && \
-  bash nodesource_setup.sh && \
-  cat /etc/apt/sources.list.d/nodesource.list
+RUN curl -sL https://deb.nodesource.com/setup_20.x | bash - && \
+    apt-get install -y nodejs
+
+COPY Pipfile Pipfile.lock /.project/
+RUN cd /.project && pipenv install --dev
+RUN pipenv install gunicorn 
 
-RUN apt-get install nodejs -y
-RUN node --version && npm --version
 
 COPY package.json /.project/package.json
 COPY package-lock.json /.project/package-lock.json
 RUN cd /.project && npm ci
-RUN mkdir -p /opt/app && cp -a /.project/. /opt/app/
 
-WORKDIR /opt/app
-
-RUN npm ci
 
+RUN mkdir -p /opt/app && cp -a /.project/. /opt/app/
+WORKDIR /opt/app
 COPY . /opt/app
 
-# build arguments
-ARG APP_ENV
-
 RUN npm run build
 
-CMD [ "npm", "start" ]
+CMD [ "sh", "-c", "npm start & pipenv run gunicorn --config gunicorn_config.py --reload server:app" ]
+
```

Comment:
I think there should be only one command needed here as `npm start` will eventually run the second statement from Makefile

Reply:
Update the changes.